docs: Added guide on combining `configure routes` and `file routes` (#12618)
remix-run/react-router: docs/start/framework/routing.md

```

> If you prefer to define your routes via file naming conventions rather than configuration, the `@react-router/fs-routes` package provides a [file system routing convention.][file-route-conventions]
> These approaches can also be combined, as shown below:

Reviewer: ```suggestion
If you prefer to define your routes via file naming conventions rather than configuration, the `@react-router/fs-routes` package provides a [file system routing convention][file-route-conventions]. Different routing conventions can even be combined:
```
